packages/@uppy/companion/src/server/controllers/send-token.js

```diff
@@ -21,10 +21,15 @@ module.exports = function sendToken (req, res, next) {
   const state = (req.session.grant || {}).state
   if (state) {
     const origin = oAuthState.getFromState(state, 'origin', req.uppy.options.secret)
+    const clientVersion = oAuthState.getFromState(
+      state,
+      'clientVersion',
+      req.uppy.options.secret
+    )
     const allowedClients = req.uppy.options.clients
     // if no preset clients then allow any client
     if (!allowedClients || hasMatch(origin, allowedClients) || hasMatch(parseUrl(origin).host, allowedClients)) {
-      return res.send(htmlContent(uppyAuthToken, origin))
+      return res.send(clientVersion ? htmlContent(uppyAuthToken, origin) : oldHtmlContent(uppyAuthToken, origin))
```

Comment:
This works but we should add a todo to the changelog to use https://www.npmjs.com/package/semver for these checks, especially if we're going to have more breaking changes.

Reply:
adding on asana so you can ignore the above